### PR TITLE
[14.0][ADD] product_category_code_unique

### PR DESCRIPTION
**Pull request by rousseldenis**
Includes:

- [ ] #844

### CODE REVIEW

**Review by ioans73 — APPROVED** [review]
Functional review

File: product_category_code_unique/data/product_category_sequence.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+  <record id="seq_product_category_auto" model="ir.sequence">
+    <field name="name">Product Category</field>
+    <field name="code">product.category</field>
+    <field eval="4" name="padding" />
+    <field name="prefix">PC/</field>
```
Comment (hparfr) [inline product_category_code_unique/data/product_category_sequence.xml]: I guess a 
`        <field name="company_id" eval="False" />`
is needed.

**Review by ajaniszewska-dev — APPROVED** [review]

File: product_category_code_unique/data/product_category_sequence.xml (hunk 1 above)
Comment (dreispt) [reply product_category_code_unique/data/product_category_sequence.xml]: @rousseldenis ?

Comment (rousseldenis) [reply product_category_code_unique/data/product_category_sequence.xml]: Indeed. Done

**Pull request merged**